edge_computing/day_2_core_cnf_clusters/telco-day-2-welcome.adoc

```diff
@@ -0,0 +1,23 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="telco-day-2-welcome"]
+= Day 2 operations for telco core CNF clusters
+include::_attributes/common-attributes.adoc[]
+:context: telco-day-2-welcome
+
+toc::[]
+
+What follows is an overview of Day 2 operations you can use to manage telco core CNF clusters.
+
+Upgrading a telco core CNF cluster:: Updating your cluster is a critical task that ensures that bugs and potential security vulnerabilities are patched. 
+For more information, see xref:../day_2_core_cnf_clusters/updating/telco-update-welcome.adoc#telco-update-welcome[Upgrading a telco core CNF cluster].
+
+Troubleshooting and maintaining telco core CNF clusters:: Troubleshooting and maintenance are weekly tasks that can be a challenge if you do not have the tools to reach your goal, whether you want to update a component or investigate an issue.
```

Comment:
I'm not sure what this first sentence is getting at, especially with this bit:
> can be a challenge if you do not have the tools to reach your goal

Can this sentence be expanded on to explain that, or maybe condensed to remove that phrase? I'm not sure exactly what to recommend, but I don't follow what the user should take away from this part.

Reply:
Honestly, I borrowed this from the target docs - which I'm not that familiar with. I get your point. I've just removed the sentence as it was ... problematic.

Reply:
haha yeah I got the sense it might have come from elsewhere. I hate getting feedback on things I didn't even write - thanks for taking it in stride :D